### PR TITLE
Lineup lab

### PR DESCRIPTION
**Pull request by darcyadams**

### CODE REVIEW

File: src/lab/Start.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import * as e6p from 'es6-promise';
+(e6p as any).polyfill();
```
Comment (andyparisi) [inline src/lab/Start.tsx]: we may actually not need this polyfill at all, since we dev using Chrome.

File: webpack/prod.js [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const nodeExternals = require('webpack-node-externals');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
+
+const config = [];
+/**
+ * Client configuration
+ */
+config.push({
```
Comment (andyparisi) [inline webpack/prod.js]: do we need a prod configuration for Lineup?

Comment (darcyadams) [reply webpack/prod.js]: no, we can delete this now

File: src/lab/Start.tsx (hunk 1 above)
Comment (darcyadams) [reply src/lab/Start.tsx]: truth, although assuming it does nothing if browser doesn't need the polyfill. Might be worth seeing if we can get this running in IE11... never know, could be useful

**Pull request merged**